gobblin-service/src/main/java/org/apache/gobblin/service/modules/flowgraph/BaseFlowGraphHelper.java

```diff
@@ -60,21 +60,21 @@
  */
 @Slf4j
 public class BaseFlowGraphHelper {
-  public static final int NODE_FILE_DEPTH = 3;
-  public static final int EDGE_FILE_DEPTH = 4;
+  private static final int NODE_FILE_DEPTH = 3;
+  private static final int EDGE_FILE_DEPTH = 4;
```

Comment:
Any reason why these depths were chosen? Could we maybe put this into a configuration so it can be easily configured if we want to edit these in the future?

Reply:
Added an example in the java docs